### ISSUE TITLE
Is there something like `jqplay` for `jtc`?

### ISSUE THREAD

**Issue by rickhg12hs** [ldn-softdev/jtc#42]
[`jqplay.org`](https://jqplay.org/) is very useful for learning `jq` and refining scripts, etc.

Is there something like that for `jtc`?

**Comment by ldn-softdev** [comment]
No, unfortunately there's none. I meant to ask to include `jtc` on the existing platforms like [https://tio.run/#](https://tio.run/#) or similar, but since I've changed my job (about +2 years back) I have literally zero bandwidth / availability to work on my pet projects.

I'll post an announcement soon asking for volunteers to take over this project so that it does not become abandonware entirely.
